app/views/dashboard/home/index.html.erb

```diff
@@ -0,0 +1,166 @@
+<div class="row">
+  <div class="col-sm-6 col-lg-4 mb-2">
+    <div class="card">
+      <div class="card-body">
+        <div class="subheader">Total Beneficiaries <i class="ti ti-address-book fs-3"></i></div>
+        <div class="d-flex align-items-baseline">
+          <div class="h1 mb-0 me-2"><%= number_with_delimiter(@dashboard_summary.beneficiaries_stats.total_count) %></div>
+          <div class="me-auto">
+            <span class="text-green d-inline-flex align-items-center lh-1">
+              <%= number_to_percentage(@dashboard_summary.beneficiaries_stats.new_count_percentage, precision: 0, strip_insignificant_zeros: true) %>
+              <i class="ti ti-trending-up icon"></i>
+            </span>
+          </div>
+        </div>
+        <div class="text-secondary mt-2"><%= number_with_delimiter(@dashboard_summary.beneficiaries_stats.new_count) %> beneficiaries added from last month</div>
```

Comment:
I think this one we can use the `pluralize` helper, passing `count` with `beneficiary` and Rails will help to put the correct pluralization